legiti/build.gradle

```diff
@@ -63,19 +63,27 @@ dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'com.snowplowanalytics:snowplow-android-tracker:1.7.1@aar'
-    implementation 'com.squareup.okhttp3:okhttp:3.12.0'
+    implementation 'com.squareup.okhttp3:okhttp:3.12.12'
     implementation 'com.google.android.gms:play-services-analytics:17.0.0'
     implementation 'com.android.support:support-annotations:28.0.0'
     implementation 'com.scottyab:rootbeer-lib:0.0.7'
-    implementation "androidx.lifecycle:lifecycle-runtime:2.2.0"
+    implementation "androidx.lifecycle:lifecycle-runtime-ktx:2.3.1"
     implementation "androidx.lifecycle:lifecycle-extensions:2.2.0"
-    annotationProcessor "androidx.lifecycle:lifecycle-compiler:2.2.0"
+    //noinspection LifecycleAnnotationProcessorWithJava8
+    annotationProcessor "androidx.lifecycle:lifecycle-compiler:2.3.1"
     testImplementation 'org.assertj:assertj-core:3.12.2'
-    testImplementation 'org.jetbrains.kotlin:kotlin-stdlib:1.3.50'
+    testImplementation 'org.jetbrains.kotlin:kotlin-stdlib:1.4.32'
     testImplementation 'junit:junit:4.12'
     testImplementation 'org.jetbrains.kotlin:kotlin-test-junit:1.3.41'
     testImplementation 'org.mockito:mockito-core:3.1.0'
     testImplementation 'org.robolectric:robolectric:4.3'
 }
 
-apply from: 'deploy.gradle'
\ No newline at end of file
+
+ext {
+    PUBLISH_GROUP_ID = 'com.legiti'
+    PUBLISH_ARTIFACT_ID = 'legiti'
+    PUBLISH_VERSION = '1.1.1'
```

Comment:
does this need to get manually updated when we publish new versions? If so, should there be a mention to it in the README?

Reply:
It's in the `DEV-GUIDE.md`